paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/resources/ResourceFolderRepository.kt

```diff
@@ -75,4 +92,19 @@ class ResourceFolderRepository(
     if (namespace != this.namespace) null else resourceTable[resourceType]
 
   override fun getNamespace(): ResourceNamespace = namespace
+
+  private class Loader(
+    private val repository: ResourceFolderRepository
+  ) : RepositoryLoader<ResourceFolderRepository>(
+    repository.resourceDir.toPath(),
+    null,
+    repository.namespace
+  ) {
+    fun load() {
+      loadRepositoryContents(repository)
+    }
+
+    override fun addResourceItem(item: BasicResourceItem, repository: ResourceFolderRepository) =
+      repository.addResourceItem(item)
+  }
```

Comment:
these look different from: https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:android/src/com/android/tools/idea/res/ResourceFolderRepository.java;l=172?q=ResourceFolderRepository&sq=

is it possible this is pointing to an earlier SHA?